### PR TITLE
[FIX] Tractogram Header in RecoBundles Tutorial

### PR DESCRIPTION
**Pull request by skoudoro**
This PR is a quick fix to address #2309.

### CODE REVIEW

File: doc/examples/bundle_extraction.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 sft_target = load_trk(target_file, "same", bbox_valid_check=False)
 target = sft_target.streamlines
-target_header = create_tractogram_header(atlas_file,
+target_header = create_tractogram_header(target_file,
                                          *sft_atlas.space_attributes)
```
Comment (BramshQamar) [inline doc/examples/bundle_extraction.py:45]: change `*sft_atlas.space_attributes ` to `*sft_target.space_attributes `

**Review by BramshQamar — COMMENTED** [review]
Hi @skoudoro 

Thanks for fixing this issue. I ran the example and it looks good. Only one change needed.

File: doc/examples/bundle_extraction.py (hunk 1 above)
Comment (skoudoro) [reply doc/examples/bundle_extraction.py:45]: good catch! will do

**Comment by skoudoro** [comment]
PR ready to go! @BramshQamar, can you have a last look?

**Comment by BramshQamar** [comment]
> PR ready to go! @BramshQamar, can you have a last look?

It is good to go! @skoudoro

**Pull request merged**